ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -98,6 +98,6 @@
 
     publishers:
       - archive:
-          artifacts: 'build/out/mgr.*.log'
+          artifacts: 'build/out/mgr.*.log, build/run/1/out/mgr.*.log, build/run/2/out/mgr.*.log'
```

Comment:
Same here.

And let's use this PR to add all logs to the API job too

Reply:
done!